src/views/studio/l10n.json

```diff
@@ -58,6 +58,7 @@
     "studio.promote": "Promote",
 
     "studio.commentsHeader": "Comments",
+    "studio.commentsNotAllowed": "Commenting for this studio has been turned off.",
```

Comment:
The message for profiles is:
> Sorry, comment posting has been turned off for this user profile. 
The message for projects is:
> Sorry, comment posting has been turned off for this project.
So the studio message here doesn't make sense. Should be something like "Sorry, comment posting has been turned off for this studio."